src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -140,70 +139,60 @@ function WorkspaceNewRoomPage(props) {
         [translate],
     );
 
-    useEffect(() => {
-        if (Permissions.canUsePolicyRooms(props.betas) && workspaceOptions.length) {
-            return;
-        }
-        Log.info('Not showing create Policy Room page since user is not on policy rooms beta');
-        Navigation.dismissModal();
-    }, [props.betas, workspaceOptions]);
-
-    if (!Permissions.canUsePolicyRooms(props.betas) || !workspaceOptions.length) {
-        return null;
-    }
-
     return (
         <ScreenWrapper
             includeSafeAreaPaddingBottom={false}
             shouldEnablePickerAvoiding={false}
         >
-            <HeaderWithBackButton title={translate('newRoomPage.newRoom')} />
-            <Form
-                formID={ONYXKEYS.FORMS.NEW_ROOM_FORM}
-                submitButtonText={translate('newRoomPage.createRoom')}
-                scrollContextEnabled
-                style={[styles.mh5, styles.flexGrow1]}
-                validate={validate}
-                onSubmit={submit}
-                enabledWhenOffline
-            >
-                <View style={styles.mb5}>
-                    <RoomNameInput
-                        inputID="roomName"
-                        autoFocus
-                        shouldDelayFocus={shouldDelayFocus}
-                    />
-                </View>
-                <View style={styles.mb2}>
-                    <Picker
-                        inputID="policyID"
-                        label={translate('workspace.common.workspace')}
-                        placeholder={{value: '', label: translate('newRoomPage.selectAWorkspace')}}
-                        items={workspaceOptions}
-                        onValueChange={setPolicyID}
-                    />
-                </View>
-                {isPolicyAdmin && (
+            <FullPageNotFoundView shouldShow={!Permissions.canUsePolicyRooms(props.betas) || !workspaceOptions.length}>
```

Comment:
Coming from https://github.com/Expensify/App/issues/27590
Since there's already a back button in RHP, there were duplicate buttons shown if `FullPageNotFoundView` is enabled
We resolved this by setting `shouldShowBackButton` prop to `false` in `FullPageNotFoundView`